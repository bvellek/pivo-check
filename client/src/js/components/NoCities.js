import React from 'react';

const NoCities = () => (
  <section className="no-cities">
    <h2>Add a City!</h2>
    <p>Looks like you haven't added any cities to your checklist. Submit above to add a city full of breweries!</p>
  </section>
);

export default NoCities;
