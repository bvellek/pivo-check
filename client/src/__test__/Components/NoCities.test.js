import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

import NoCities from '../../js/components/NoCities';

describe('<NoCities />', () => {
  it('renders without crashing', () => {
    shallow(<NoCities />);
  });
  it('it renders correctly', () => {
    const component = shallow(
      <NoCities />
    );
    expect(shallowToJson(component)).toMatchSnapshot();
  });
});
