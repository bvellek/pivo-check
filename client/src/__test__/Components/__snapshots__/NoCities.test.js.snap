// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NoCities /> it renders correctly 1`] = `
<section
  className="no-cities"
>
  <h2>
    Add a City!
  </h2>
  <p>
    Looks like you haven't added any cities to your checklist. Submit above to add a city full of breweries!
  </p>
</section>
`;
